group_vars/maas_region_controller/10-maas_region_controller

```diff
@@ -9,6 +9,8 @@ admin_password: "admin"
 admin_email: "admin@email.com"
 admin_id: "admin"
```

Comment:
Looking at this again we could remove this and have no default import?

Reply:
I've just seen that myself, removing the default and checking for whether the variable was defined is a much cleaner way to achieve the same results